apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -15,6 +15,11 @@
 package v1alpha1
 
 import (
+
+	// nolint:gosec // we don't use md5 in the context of cryptography
+	"crypto/md5"
```

Comment:
Can we use sha256 instead of md5?

Reply:
e can, but is there any benefit to it? In fact, we could directly use the
first few bytes of the cluster ID if it were guaranteed to be a UUID.

Reply:
Got it. However, I would avoid using md5 and all those nolint instructions.